src/cmake/externalpackages.cmake

```diff
@@ -113,7 +113,9 @@ else ()
 endif()
 
 # From pythonutils.cmake
-find_python()
+if (USE_PYTHON)
```

Comment:
Is there any particular reason not to use the existing `if (USE_PYTHON)` conditional 3 lines below this?

Reply:
Yes, because find_python can turn off USE_PYTHON if not found.

Reply:
Ah, sneaky, I see.